crmd/notify.c

```diff
@@ -20,120 +20,501 @@
 #include <crm/crm.h>
 #include <crm/msg_xml.h>
 #include "notify.h"
+#include "crmd_messages.h"
 
 char *notify_script = NULL;
 char *notify_target = NULL;
+GListPtr notify_list = NULL;
+
+typedef struct {
+    char *id;
+    char *path;
+    int timeout;
+    char *tstamp_format;
+    char *info;
```

Comment:
For the first paragraph I consider it rather error-prone to copy around scripts and have multiple versions of them which just differ in a string somewhere - and of course and that is the dangerous part the fixes that happen to just a subset of them. On a production-cluster I would rather distribute generic-scripts in rpms than copy something somewhere in the filesystem.

For the second paragraph - this sounds interesting and I will invest further thinking in a
more generic approach.